src/Package/MSBuild.Engine.Corext/MsBuild.Engine.Corext.nuspec

```diff
@@ -32,6 +32,7 @@
     <file src="$X86BinPath$/System.Memory.dll" target="v15.0/bin" />
     <file src="$X86BinPath$/System.Text.Json.dll" target="v15.0/bin" />
     <file src="$X86BinPath$/Microsoft.Bcl.AsyncInterfaces.dll" target="v15.0/bin" />
+    <file src="$X86BinPath$/Microsoft.Extensions.ObjectPool.dll" target="v15.0/bin" />
```

Comment:
Don't you also have to add the StringTools assembly in here?

Reply:
Thanks, fixed! I have yet to verify the whole thing with an exp insertion. It is likely that more packaging-related issues will surface.